examples/multiphysics/3d-dam-break/3d-dam-break_postprocess.py

```diff
@@ -0,0 +1,179 @@
+"""
```

Comment:
If you want to use the Lethe_pyvista_tools module to do some cleaning here we could try it. We can do it together if you want. Otherwise, it is good as is.

Reply:
I think for this PR this is good. :) We can revisit this during the Hackathon :)